pages/atoms/divider-icon/divider.tsx

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+import { NavigationData } from "../../../data";
+
+export default function Divider(props: NavigationData) {
+  return (
+    <div role="divider">
```

Comment:
Im not sure if adding a role would be best here, roles are typically announced to screen readers and we might not want to have this announced, try to get image by alt tag/text